webui/react/src/pages/ExperimentDetails.tsx

```diff
@@ -28,23 +35,29 @@ interface Params {
 }
 
 const ExperimentDetailsComp: React.FC = () => {
-  const { experimentId: experimentIdParam } = useParams<Params>();
-  const experimentId = parseInt(experimentIdParam);
-  const [ experiment, setExpRequestParams ] =
```

Comment:
nit: IMO `set...Params` , although not consistent with the rest of the code base, is more accurate, it's not making the request and just setting the params and triggering a request because of a hook dependency change.

Reply:
you can't await on this "request"